.buildpacks

```diff
@@ -0,0 +1,2 @@
+https://github.com/heroku/heroku-buildpack-nodejs.git
+https://github.com/heroku/heroku-buildpack-python.git
```

Comment:
had to use a custom heroku buildpack so it knows to install npm dependencies (bower) before running the application